src/definition/scheduler/IRecurringSchedule.ts

```diff
@@ -8,6 +8,10 @@ export interface IRecurringSchedule {
      * or a number.
      */
     interval: string | number;
+    /**
+     * Whether to let the first iteration to execute as soon as the task is registered
```

Comment:
Bit of rewording, wdyt?

```suggestion
     * Whether the recurring job should start immediately or wait for the interval
```